NOTICE

```diff
@@ -47,6 +44,106 @@ types-urllib3 1.26.15 : Apache License 2.0
 urllib3 1.26.9 : MIT License
 zipp 3.6.0 : MIT License
 
+
+Copyright Text: 
+
+Click - Python Command Line Utility 8.0.4 pypi:click/8.0.4
+	Copyright 2002-2006 Python Software Foundation. All rights reserved
+	Copyright 2001-2006 Gregory P. Ward
+	Copyright 2001-2006 Gregory P. Ward. All rights reserved
+	Copyright 2002-2006 Python Software Foundationimport typing as tfrom collections import dequefrom gettext import gettext as _from gettext import ngettext
+CommonMark-py 0.9.1 pypi:commonmark/0.9.1
+	Copyright (c) 2014, John MacFarlaneAll rights reserved
+	Copyright (c) 2014, Bibek Kafle and Roland Shoemaker
+Packaging 21.3 pypi:packaging/21.3
+	copyright = about
+PyYAML 5.4.1 pypi:PyYAML/5.4.1
+	Copyright (c) 2017-2021 Ingy d
+	Copyright (c) 2006-2016 Kirill Simonov
+Pyflakes 2.4.0 pypi:pyflakes/2.4.0
+	Copyright 2005-2011 Divmod, Inc.
+	Copyright 2013 Florent Xicluna.  See LICENSE file for detailsfrom __future__ import with_statement
+	Copyright 2013-2014 Florent Xicluna
+RonnyPfannschmidt/iniconfig 1.1.1 pypi:iniconfig/1.1.1
+	(C) Ronny Pfannschmidt, Holger Krekel -- MIT licensed"""__all__ = ['IniConfig', 'ParseError']
+atomicwrites 1.4.0 pypi:atomicwrites/1.4.0
+	copyright = '2015, Markus Unterwaditzer
+	Copyright (c) 2015-2016 Markus Unterwaditzer
+	copyright = '2015-2017, Markus Unterwaditzer
+charset-normalizer 2.0.12 pypi:charset-normalizer/2.0.12
+	Copyright (c) 2019 TAHRI Ahmed R.
+	copyright: (c) 2021 by Ahmed TAHRI
+	copyright = '2019, Ahmed TAHRIauthor = 'Ahmed TAHRI @Ousret'
+dataclasses 0.8 pypi:dataclasses/0.8
+	(C)        with self.assertRaisesRegex(TypeError,            asdict(int)
+	(C)        with self.assertRaisesRegex(TypeError,            astuple(int)
+	(C)        # the_fields is a tuple of 3 items, each value        #  is in __annotations__.        self.assertIsInstance(the_fields, tuple)        for f in the_fields:
+	(C)        with self.assertRaisesRegex(TypeError,            fields(C())

```

Comment:
Contains improper copyright attributions.

Reply:
Removed. And I added the "author" from here as copyright: https://pypi.org/project/dataclasses/

Reply:
LGTM

Reply:
I did find this: `Copyright 2017-2022 Eric V. Smith, all rights reserved.` in [README.rst](https://github.com/ericvsmith/dataclasses/blob/5f6568c3468f872e8f447dc20666628387786397/README.rst). Perhaps use that as-is?

Reply:
Added